res/translations/es.json

```diff
@@ -0,0 +1,230 @@
+{
+    "A new version is available": "Una nueva versión está disponible",
+    "About": "Acerca de",
```

Comment:
"About" is used in the gear button menu to bring up a dialog with information about the app. I'm guessing translating it more like "Info" may be closer in meaning?

Reply:
This is a case where the literal translation was used initially and stuck. Happy to change it to `Información adicional` but it makes more sense to keep IMO. See the image below for instance
![Captura-de-pantalla-2022-05-12-a-las-13 48 50](https://github.com/user-attachments/assets/97be524a-1233-4382-86cb-d8275dfc9588)